src/api/requests.js

```diff
@@ -358,100 +274,50 @@ export const requests = {
+      ...filters,
+      order: 'desc',
+      limit: store.getters.pageLimit
+    })
+    return response
   },
 
-  getAssetRequests (filters) {
+  async getAssetRequests (filters) {
     const params = clearObject({
       asset: filters.code,
       requestor: filters.requestor,
       state: filters.state,
       reviewer: filters.reviewer,
       order: 'desc'
     })
-
-    return new ScopedServerCallBuilder()
-      .request()
-      .assets()
-      .sign()
-      .get(params)
+    return (await Sdk.horizon.request.getAllForAssets(params))
   },
 
-  getLimitsUpdateRequests ({ asset, state, requestor }) {
+  async getLimitsUpdateRequests ({ asset, state, requestor }) {
     const filters = {}
     if (asset) filters.dest_asset_code = asset
     if (state) filters.state = state
     if (requestor) filters.requestor = requestor
-
-    return new ScopedServerCallBuilder()
-      .request()
-      .limits_updates()
-      .sign()
-      .get({
-        order: 'desc',
-        limit: store.getters.pageLimit,
-        ...filters
-      })
+    return (await Sdk.horizon.request.getAllForLimitsUpdates({
+      order: 'desc',
+      limit: store.getters.pageLimit,
+      ...filters
+    }))
   },
 
   // legacy
 
-  reviewRequest (opts) {
-    opts.source = config.MASTER_ACCOUNT
-    const operation = ReviewRequestBuilder.reviewRequest(opts)
-    return server.submitOperation(operation, true)
-  },
-
-  getPreissuanceRequests (asset) {
+  async getPreissuanceRequests (asset) {
     if (asset.toLowerCase() === 'all') {
       asset = ''
     }
-
-    return server.sdkServer.reviewableRequestsHelper()
-      .preissuances()
-      .forAssetCode(asset)
-      .order('desc')
-      .forReviewer(config.MASTER_ACCOUNT)
-      .limit(store.getters.pageLimit)
-      .callWithSignature(store.getters.keypair)
-      .then(response => {
-        response.records = mapRequests(response.records)
-        return response
-      })
-  },
-
-  getAssetCreationRequests ({ state, requestor, asset }) {
-    return server.sdkServer.reviewableRequestsHelper()
-      .assets()
-      .order('desc')
-      .forReviewer(config.MASTER_ACCOUNT)
-      .limit(store.getters.pageLimit)
-      .forState(state)
-      .forRequestor(requestor)
-      .forAssetCode(asset)
-      .callWithSignature(store.getters.keypair)
-      .then(response => {
-        response.records = mapRequests(response.records)
-        return Promise.resolve(response)
-      })
-  },
-
-  getTokenRequestById (id) {
-    return server.sdkServer.reviewableRequestsHelper()
-      .assets()
-      .reviewableRequest(id)
-      .callWithSignature(store.getters.keypair)
-      .then(response => {
-        return Promise.resolve(new TokenRequest(response))
-      })
+    const response = await Sdk.horizon.request.getAllForPreissuances({
+      asset: asset,
+      order: 'desc',
+      reviewer: config.MASTER_ACCOUNT
+    })
+    response.records = mapRequests(response.data.records)
```

Comment:
Just checked, we're using `mapRecords` only in this place and will always return us the `CreatePreIssuanceRequest` instance . That means:

 * we can safely remove any request class except `CreatePreIssuanceRequest`
 * at the end of the chain there is a `PreIssuanceRequestList` component, it wouldn't take a lot of effort to rewrite it without the `CreatePreIssuanceRequest` and totally remove this stuff